src/context/AuthContext.tsx

```diff
@@ -71,7 +71,7 @@ export const AuthProvider: ChildrenFC = ({ children }) => {
     setUser(emptyUser);
   }, []);
 
-  const isUserLoggedIn = user.id !== -1;
+  const isUserLoggedIn = user.id !== '-1';
```

Comment:
What about making the `id` and the `user` empty when not logged in instead of relying on some arbitrary value? Wouldnt that be safer (and easier) ?